integration/walletextension/wallet_extension_test.go

```diff
@@ -300,31 +301,6 @@ func TestCannotSubmitTxFromAnotherAddressAfterSubmittingViewingKey(t *testing.T)
 	}
 }
 
-// TODO - #453 - Build out this test as we expand the subscription functionality.
-func TestCanSubscribeForLogs(t *testing.T) {
```

Comment:
I removed this test for now, as it no longer worked (since attempts to subscribe via HTTP are now rejected, as they should be). I added the test below (subscribing over HTTP should fail), and I'll readd this test in a future PR, as part of the events work.